packages/conf-lld/conf-lld.1/opam

```diff
@@ -0,0 +1,29 @@
+opam-version: "2.0"
+maintainer: "Léo Andrès <contact@ndrs.fr>"
+authors: "The LLVM team"
+homepage: "http://llvm.org"
+bug-reports: "https://llvm.org/bugs/"
+license: "MIT"
+build: [
+  ["wasm-ld" "-version"] { os-distribution != "homebrew" }
```

Comment:
I'm not sure how to properly test this in homebrew.

Basically, the executable is installed but not in the PATH. One need to do something like `export PATH="$(brew --prefix llvm@15)/bin/:$PATH"` but then, it'll only contains `lld` which is a generic driver. One should rather invoke `wasm-ld` but I have no idea where it is located and how to find it.

I would say it's good enough in the current state and can be improved later?

Reply:
I think it's ok although I'm also at a loss regarding homebrew.

Hopefully a homebrew user will find the time to improve things.